packages/e2e-tests/tests/object_preview-01.test.ts

```diff
@@ -3,137 +3,80 @@ import {
+  await waitForConsoleMessage(screen, "(20) [0, 1, 2, 3, 4, …]");
+  await waitForConsoleMessage(screen, "Uint8Array(20) [0, 1, 2, 3, 4, …]");
+  await waitForConsoleMessage(screen, "Set(22) [{…}, {…}, 0, 1, 2, …]");
 
-  await waitForConsoleMessage(screen, "Map(21) { {…} → {…}, 0 → 1, 1 → 2, 2 → 3, 3 → 4, 4 → 5,");
-  await waitForConsoleMessage(screen, "WeakSet(20) [ {…}, {…}, {…},");
-  await waitForConsoleMessage(screen, "WeakMap(20) { {…} → {…}, {…} → {…},");
-  await waitForConsoleMessage(screen, "Object { a: 0, a0: 0, a1: 1, a2: 2, a3: 3, a4: 4,");
-  await waitForConsoleMessage(screen, "/abc/gi");
-  await waitForConsoleMessage(screen, "Date");
-
-  await waitForConsoleMessage(screen, `RangeError: "foo"`);
+  await waitForConsoleMessage(screen, "Map(21) {{…} → {…}, 0 → 1, 1 → 2, 2 → 3, 3 → 4, …}");
+  await waitForConsoleMessage(screen, "WeakSet(20) [{…}, {…}, {…}, {…}, {…}, …]");
   await waitForConsoleMessage(
     screen,
-    '<div id="foo" class="bar" style="visibility: visible" blahblah="">'
+    "WeakMap(20) {{…} → {…}, {…} → {…}, {…} → {…}, {…} → {…}, {…} → {…}, …}"
   );
+  await waitForConsoleMessage(screen, "{a: 0, a0: 0, a1: 1, a2: 2, a3: 3, …}");
+  await waitForConsoleMessage(screen, "/abc/gi");
+  await waitForConsoleMessage(screen, "Sun Aug 14 2022 12:23:25 GMT-0700 (Pacific Daylight Time)");
 
-  await waitForConsoleMessage(screen, "function bar()");
-  await checkJumpIcon(screen, "function bar()");
-  // await new Promise(r => setTimeout(r, 100_000));
+  await waitForConsoleMessage(screen, `RangeError: foo`);
 
-  await waitForConsoleMessage(screen, 'Array(6) [ undefined, true, 3, null, "z", 40n ]');
-  await waitForConsoleMessage(screen, "Proxy {  }");
+  await waitForConsoleMessage(screen, "ƒbar()");
+
+  await waitForConsoleMessage(screen, "Proxy{}");
   await waitForConsoleMessage(screen, "Symbol()");
   await waitForConsoleMessage(screen, "Symbol(symbol)");
-  await waitForConsoleMessage(screen, `Object { "Symbol()": 42, "Symbol(symbol)": "Symbol()" }`);
-
-  msg = await waitForConsoleMessage(screen, "Object { _foo: C }");
+  await waitForConsoleMessage(screen, `{Symbol(): 42, Symbol(symbol): Symbol()}`);
 
-  // TODO: add support for expanding objects in the console
-  await checkMessageObjectContents(
-    screen,
-    msg,
-    ['foo: C { baz: "baz" }', 'bar: "bar"', 'baz: "baz"'],
-    ["foo", "bar"]
-  );
+  await waitForConsoleMessage(screen, "{_foo: C{…}, foo: ƒ()}");
 
   await warpToMessage(screen, "Done");
 
   await executeInConsole(screen, "Error('helo')");
-  await waitForConsoleMessage(screen, 'Error: "helo"');
+  await waitForConsoleMessage(screen, "Error: helo");
 
-  await executeInConsole(
-    screen,
-    `
-        function f() {
-          throw Error("there");
-        }
-        f();
-      `
-  );
+  await executeInConsole(screen, 'function f() { throw Error("there"); }()');
   // FIXME the first line in this stack isn't right.
```

Comment:
Is this FIXME still relevant? seems like it isn't.

Reply:
I wasn't sure, deleting as the output seems correct.